examples/plugin/CMakeLists.txt

```diff
@@ -8,3 +8,10 @@
 raja_add_executable(
   NAME plugin-example
   SOURCES test-plugin.cpp counter-plugin.cpp)
+
+raja_add_executable(
+  NAME plugin-example-dynamic
+  SOURCES test-plugin-dynamic.cpp)
+  
+add_library(timer_plugin SHARED timer-plugin.cpp)
+target_link_libraries(timer_plugin PRIVATE RAJA)
```

Comment:
@davidbeckingsale this `target_link_libraries` call with `PRIVATE RAJA` was needed due to needing camp/RAJA includes. I realize this is tangential, but do you think there is a way we could create an include-only RAJA target that won't automatically link `libRAJA.a` ?

Reply:
That's not a bad idea, we could get them out of the target by just requesting the includes I suppose, but we probably want everything but the actual link there anyway.